### PR TITLE
Activation function configuration

### PR DESCRIPTION
**Pull request by by-dam**
Configure activation function for each layer in the model's layer.txt file:

```
1200 250 100 3
ReLU ReLU ReLU Sigm
```

Possible values for the activation function are:
* `ReLU` for Rectified Linear Unit
* `Sigm` for Logistic Sigmoïd
* `Tanh` for Hyperbolic Tangent
* `Iden` for Identity function

### CODE REVIEW

File: MuvrKitTests/MKForwardPropagatorTests.swift [hunk 1]
```diff
@@ -62,9 +67,13 @@ class MKForwardPropagatorTests : XCTestCase {
     /// (NAND) matrix model test using ReLU
     ///
     func testModelOfNANDMatrix() {
-        var conf = baseConfiguration
-        conf.hiddenActivation = ReLUActivation()
-        conf.outputActivation = ReLUActivation()
+        let conf = MKForwardPropagatorConfiguration(
+            layerConfiguration: [
+                MKForwardPropagatorLayerConfiguration(elements: 2, activation: ReLUActivation()),
+                MKForwardPropagatorLayerConfiguration(elements: 1, activation: SigmoidActivation())
```
Comment (tmbo) [inline MuvrKitTests/MKForwardPropagatorTests.swift]: You did change the output activation here. Doesn't that break the test?

File: Muvr/Models.bundle/slacking_model.layers.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-1200 500 100 25 2
\ No newline at end of file
+1200 500 100 25 2
+tanh tanh tanh tanh Sigm
```
Comment (tmbo) [inline Muvr/Models.bundle/slacking_model.layers.txt]: Did you update the model? The model I trained the other week was not trained with `tanh`.

**Comment by janm399** [comment]
Great minds work alike: I've hacked the same thing on the plane this morning; see #92.

**Comment by by-dam** [comment]
Not needed anymore

**Pull request closed without merging**